content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,137 @@
+
+1. Append new CA to the file against `--client-ca-file` flag in the `kube-apiserver` and `kube-scheduler` configuration.
+
+    {{< note >}}
+
+    If you used `kubeadm` to deploy your cluster, the control plane components run as {{< glossary_tooltip text="static Pods" term_id="static-pod" >}}, managed by the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}.
+
+    {{< /note >}}
+
+1. Update certificates for user accounts by replacing the content of `client-certificate-data` and `client-key-data` respectively and the `certificate-authority-data` section in all `admin.conf`, `controller-manager.conf`, `scheduler.conf` files with Base64-encoded new certificate data:
+
+    For information about creating certificates for individual user accounts, see [Configure certificates for user accounts](/docs/setup/best-practices/certificates/#configure-certificates-for-user-accounts).
+
+1. Restart API servers with the certificates (`apiserver.crt`, `apiserver-kubelet-client.crt`, and `front-proxy-client.crt`) signed by new CA. You can use the existing private keys or new private keys. If you changed the private keys then update these in the Kubernetes certificates directory as well.
+
+    Since the pod trusts both old + new CAs, there will be a momentarily disconnection after which the pod's kube client will reconnect to the new API server that uses the certificate signed by the new CA.
+
+    *    Restart Scheduler to use the new CAs.
+
+    *    Make sure control plane components logs no TLS errors.
+
+    {{< note >}}
+
+    To generate certificates and private keys for your cluster using the `openssl` command line tool, see [Certificates (`openssl`)](/docs/concepts/cluster-administration/certificates/#openssl).    
+    You can also use [`cfssl`](/docs/concepts/cluster-administration/certificates/#cfssl).
+
+    {{< /note >}}
+
+1. If your cluster is using bootstrap tokens to join nodes, update the ConfigMap `cluster-info` in the `kube-public` namespace with new CA.
+
+    ```shell
+    base64_encoded_ca="$(base64 /etc/kubernetes/pki/ca.crt)"
+
+    kubectl get cm/cluster-info --namespace kube-public -o yaml | \
+        /bin/sed "s/\(certificate-authority-data:\).*/\1 ${base64_encoded_ca}" | \
+        kubectl apply -f -
+    ```
+
+1. Update the file against `clientCAFile` in kubelet configuration and `certificate-authority-data` in kubelet.conf to use the new CA on all nodes.
+
+    If your kubelet is not using client certificate rotation update `client-certificate-data` and `client-key-data` in kubelet.conf on all nodes along with the kubelet client certificate file usually found in `/var/lib/kubelet/pki`.
+
+1. Follow these steps on all nodes in a rolling fashion.
+
+    1. Restart kubelet to start using the new CA
+
+    1. Restart any other *[aggregated api servers](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/)* or *webhook handlers* to trust the new CA certificates.
```

Comment:
I think this step and the following step could be pulled out into top level steps. You wouldn't restart all pods for ever _N_ nodes in your cluster

Reply:
agreed and moved.